### PR TITLE
Remove unused methods

### PR DESCRIPTION
**Pull request by reorder**

### CODE REVIEW

File: src/miner.cpp [hunk 1]
```diff
@@ -368,41 +368,6 @@ void IncrementExtraNonce(CBlock* pblock, CBlockIndex* pindexPrev, unsigned int&
 double dHashesPerSec = 0.0;
 int64_t nHPSTimerStart = 0;
 
-//
```
Comment (btcdrak) [inline src/miner.cpp:371]: can we `#if 0` this codeblock instead?

**Comment by reorder** [comment]
Makes no sense: all other code for solo mining is removed upstream, this code has no other use.

**Comment by btcdrak** [comment]
OK.  utACK

**Comment by btcdrak** [comment]
Merged in 076b3118d08c0cd5d02a0fad207fb895b064d3ce

**Pull request closed without merging**